test/app/defaults.test.js

```diff
@@ -14,7 +14,7 @@ describe('Chisel Generator with default options', function () {
         'skip-install': true
       })
       .withPrompts({
-        name: 'Test Project',
+        name: 'Test Project1',
```

Comment:
Would be nice to test more cases, e.g. with non-latin characters, like @jakub300 mentioned